Aspen-Discovery/aspen-discovery
# Activate aspen admin searchbar

Clicking 'Search' in the 'Administration Options' sidebar automatically places the cursor inside the search bar,  which makes for a more seamless page navigation.

Test plan:

 - log in to Aspen as an admin
 - open any Aspen Administration section so that the Administration Options sidebar shows
 - Click 'Search'
    -> if the patch is applied, the input field is immediately focused on
    -> without the patch, the input field is inactive at first, and only focused on if it is then manually selected by the user
    
Note: this seems to fail the tabs vs space QA. While the changes do not introduce any spaces used instead of tabs, there appear to be some in other parts of the modified files. Should this PR also clean these up in a separate 'style' commit?
